mail_chatter_company_tracking/models/mail_message.py

```diff
@@ -0,0 +1,31 @@
+from odoo import models
+
+
+class Message(models.Model):
+    _inherit = "mail.message"
+
+    def _message_format(self, fnames):
+        vals_list = super()._message_format(fnames)
+        allowed_companies = self.env.context.get("allowed_company_ids", [])
```

Comment:
I don't think filtering the msg is a good idea. Mainly because the next case:
I have the next product fields as trackeable:
- Barcode (No company dependant)
- Cost (Company dependant)

Steps:
1. Modify product barcode and cost on company1
2. The msg appear properly
3. ![image](https://github.com/user-attachments/assets/33a5107e-281f-467f-89e3-7d2d6a3237cc)
4. If i change to company2 the msg is not visible, but the barcode is shared, so I should be able to see the change

Also this display doesnt seems right as the barcode is not company dependant:
![image](https://github.com/user-attachments/assets/e794a58f-b7be-4aa0-ab1f-b84c1f3de90e)

I think you could or just display the company where the change was made at field level (when is company dependant) or move the filtering at line level instead of the hole msg.

Let me know what you think about it.